src/dynamic_boundary_conditions/rainfall_data_from_hirds.py

```diff
@@ -111,14 +112,22 @@ def get_layout_structure_of_data(site_data: str) -> List[Tuple[int, float, str]]
                 rcp.append(float(rcp_result[0]))
                 time_period.append(period_result[0])
             else:
-                rcp.append(float('nan'))
+                rcp.append(float("nan"))
```

Comment:
Could we have a comment here to specify what the `NaN` represents?

Reply:
added a comment